<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinaryOptionsWidget</class>
 <widget class="QWidget" name="BinaryOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>177</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lVectors">
     <property name="text">
      <string>Vectors:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KIntNumInput" name="niVectors">
     <property name="toolTip">
      <string>The number of vectors in the file</string>
     </property>
     <property name="value">
      <number>2</number>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lFormat">
     <property name="text">
      <string>Data type:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KComboBox" name="cbDataType"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lByteOrder">
     <property name="text">
      <string>Byte order:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KComboBox" name="cbByteOrder"/>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>74</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
